src/js/constants.js

```diff
@@ -84,6 +84,7 @@ export const MediaUIAttributes = {
   MEDIA_LOADING: 'media-loading',
   MEDIA_BUFFERED: 'media-buffered',
   MEDIA_STREAM_TYPE: 'media-stream-type',
+  MEDIA_TARGET_LIVE_WINDOW: 'media-target-live-window',
```

Comment:
I don't believe so. I think "target live window" makes sense still in our "DVR" proposal. We can always amend it if necessary once we move https://github.com/video-dev/media-ui-extensions/issues/4 to an official proposal PR, since there isn't a custom/extended media element that supports this yet (and `mux-video` will be the first, so we can treat it as "internal")